packages/flame_riverpod/README.md

```diff
@@ -0,0 +1,76 @@
+# flame_riverpod
+
+Riverpod is a reactive caching and data-binding framework for Dart & Flutter.
+
+In `flutter_riverpod`, widgets can be configured to rebuild when the state
+of a provider changes.
+
+When using Flame, we are interacting with components, which are *not* Widgets.
+
+`flame_riverpod` provides the `ComponentRef` and `HasComponentRef` to
+facilitate managing state from Providers in your Flame Game.
+
+
+## Usage
+
+Your Widget that extends `FlameGame` should use the HasComponentRef mixin,
```

Comment:
technically the game (that extends FlameGame) is not itself a widget -- it must be wrapped by a GameWidget for that